.github/workflows/regression-suites.yml

```diff
@@ -14,29 +15,27 @@ jobs:
       fail-fast: false
       matrix:
         platform:
-          - os: windows-latest
```

Comment:
I think we still want to run the regression on windows, right?

Reply:
oh yeah, definitely. will fix 👍